backend/__test__/filtersTests/datafileBooleanFilters.test.ts

```diff
@@ -0,0 +1,219 @@
+import request from "supertest";
+import { expect, describe, it, afterAll, beforeAll } from "@jest/globals";
+import App from "../../src/app";
+import mongoose from "mongoose";
+import { MongoMemoryServer } from "mongodb-memory-server";
+import { checkArrayContainsObjects } from "../utils/helpers";
+import { Application } from "express";
+
+let app: Application;
+let mongoServer: MongoMemoryServer;
+
+beforeAll(async () => {
+  // Create MongoDB
+  mongoServer = await MongoMemoryServer.create();
+  await mongoose.connect(mongoServer.getUri());
+  app = new App().express;
+  // Post documents
+  await request(app).post("/api/datafiles").send(document1);
+  await request(app).post("/api/datafiles").send(document2);
+  await request(app).post("/api/datafiles").send(document3);
+  await request(app).post("/api/datafiles").send(document4);
+  await request(app).post("/api/datafiles").send(document5);
```

Comment:
I think it's better if we add the documents manually to the DB and not through our app, as with this test we only want to test the filter and not the CRUD-operation.

Reply:
Fixed.